pyanaconda/modules/storage/partitioning/base.py

```diff
@@ -67,6 +72,25 @@ def on_selected_disks_changed(self, selection):
         """Keep the current disk selection."""
         self._selected_disks = selection
 
+    def get_device_tree(self):
+        """Get the device tree.
+
+        :return: a DBus path to a device tree
+        """
+        if not self._device_tree_module:
+            module = DeviceTreeModule()
+            module.on_storage_reset(self.storage)
+            self._device_tree_module = module
+
+        if not self._device_tree_path:
+            self._device_tree_path = publish_device_tree(
+                message_bus=DBus,
+                namespace=STORAGE.namespace,
+                device_tree=self._device_tree_module
+            )
+
+        return self._device_tree_path
```

Comment:
What about making this a property `device_tree`?

Reply:
Properties have signals and other synchronization. I don't want to create this module until someone really asks for it.

Reply:
I meant "private" property not part of the module API. That should be fine I guess.

Reply:
I don't think that this property could be useful to the python module, since the method returns a DBus path. I will probably eventually create a property for the `self._device_tree_module`, but it is not necessary at this moment.

Reply:
Ohh, OK, I thought it's an object not a path.